.github/workflows/test.yaml

```diff
@@ -146,6 +146,46 @@ jobs:
         name: coverage
         path: coverage.enterprisepostgres.out
 
+  integration-tests-enterprise-postgres-nightly:
+    environment: "Configure ci"
+    runs-on: ubuntu-latest
+    needs: runnable
+    if: "needs.runnable.outputs.runnable == 'true'"
+    steps:
+
+    - name: setup golang
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
+
+    - name: cache go modules
+      uses: actions/cache@v3
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+
+    - name: checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: run integration tests
+      run: make test.integration.enterprise.postgres
+      env:
+        KONG_LICENSE_DATA: ${{ secrets.KONG_LICENSE_DATA }}
```

Comment:
This is not set in the repo/env secrets anymore after https://github.com/Kong/kubernetes-ingress-controller/pull/2648 got merged. If you take a look at the changes it introduced it's gonna be fairly easy to deduce what step to add here to pull the license in.